app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml

```diff
@@ -62,6 +62,9 @@
             </block>
         </referenceContainer>
 
+        <move element="promotion-block" destination="sidebar.main" after="-"/>
```

Comment:
you can add your `promotion-block` to the `sidebar.main` initially and not use `move` instruction

Reply:
this was a test! I deleted this